crates/storage_models/src/merchant_connector_account.rs

```diff
@@ -54,6 +54,8 @@ pub struct MerchantConnectorAccountNew {
     pub business_country: storage_enums::CountryCode,
     pub business_label: String,
     pub business_sub_label: Option<String>,
+    pub created_at: Option<time::PrimitiveDateTime>,
+    pub modified_at: Option<time::PrimitiveDateTime>,
```

Comment:
You can have these as required values for insertion, since they must always be provided, or the database would throw errors. The existing implementation would still allow the possibility of not passing any timestamps for the `created_at` and `modified_at` fields.

Reply:
Since `MerchantConnectorAccountNew` implements `Default` trait through `#[derive()] ` macro, every data member of the `MerchantConnectorAccountNew` struct needs to implement `Default` Macro. But `time::PrimitiveDateTime `doesn't implement it. 

But `Option<time::PrimitiveDateTime>` would defaults to None.

We would need to manually implement Default trait and for MerchantConnectorAccountNew and provide implementation for the `Default::default()` function.
[https://stackoverflow.com/questions/68346169/is-there-a-short-way-to-implement-default-for-a-struct-that-has-a-field-that-doe](url)

Reply:
PaymentIntentNew also has `Option<time::PrimitiveDateTime>` for `created_at` and `modified_at` .

Reply:
From what I can find, we do not use the `Default` implementation for `MerchantConnectorAccountNew` anywhere in our codebase, and in the only place where a `MerchantConnectorAccountNew` is constructed (in `core/admin.rs`), all fields are specified explicitly. It should be safe to remove the `#[derive(Default)]` annotation on `MerchantConnectorAccountNew`.